src/kirby/components/grid/grid.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit, Input, OnDestroy } from '@angular/core';
+import { Component, OnInit, Input, OnDestroy, HostBinding } from '@angular/core';
```

Comment:
:fire: Remove obsolete import:
```suggestion
import { Component, OnInit, Input, OnDestroy } from '@angular/core';
```